.github/workflows/test.yaml

```diff
@@ -41,10 +43,7 @@ jobs:
 
       - name: Test
         env:
-          IGN_DB_USERNAME: ${{ job.services.mysql.MYSQL_USER }}
-          IGN_DB_PASSWORD: ${{ job.services.mysql.MYSQL_PASSWORD }}
-          IGN_DB_ADDRESS: 127.0.0.1
-          IGN_DB_NAME: ${{ job.services.mysql.MYSQL_DATABASE }}
+          IGN_DB_ADDRESS: "127.0.0.1:${{ job.services.mysql.ports[3306] }}"
         run: |
-          go test -race -covermode=atomic -coverprofile=coverage.tx -v ./...
```

Comment:
Having race conditions here made the test execution longer, I understand it's important to keep race condition checking, but we might need to evaluate other alternatives, like running it in the `main` branch.

Reply:
How much time is the job allowed to run? 

A quick hack to make this quicker: Would it make sense to run `go test -race` for different packages in different jobs? We would parallelize the work making it faster, and it would also reduce the amount of time each single job will need to run.

Reply:
> For how long is the job allowed to run?

60m.

> A quick hack to make this quicker: Would it make sense to run go test -race for different packages in different jobs? We would parallelize the work making it faster, and it would also reduce the amount of time each single job will need to run.

Would it make sense to consider doing this in another PR? I can create a follow up task.

Reply:
Can I get your eyes here? @AlejoAsd

Reply:
> Would it make sense to consider doing this in another PR? I can create a follow up task.

Definitely. I answered this as a potential issue to the question you asked. I'm okay doing it here or in another PR.

Reply:
Issue added here: #10